src/routes/(admin sponsor)/sponsor-admin/my-network/+page.server.js

```diff
@@ -0,0 +1,17 @@
+import lodash from 'lodash';
```

Comment:
we should prolly add a comment above stating that without page.server security won't fire... aka, future me, please don't refactor and delete.